nifi-commons/nifi-utils/src/test/java/org/apache/nifi/util/validator/TestStandardValidators.java

```diff
@@ -288,4 +288,42 @@ public void testiso8061InstantValidator() {
         vr = val.validate("foo", "2016-01-01T01:01:01.000Z", vc);
         assertTrue(vr.isValid());
     }
+
```

Comment:
This would be a great use case for Groovy instead of Java. I've started doing that with my unit tests because you can specify the JSON like this:

```
import static groovy.json.JsonOutput.*

def json = prettyPrint(toJson([
    Name: "Crockford, Douglas"
]))
```

Not required, but worth thinking about because it's a lot cleaner and Groovy is allowed in tests.

Reply:
Looks interesting. I'll try to make use of this in my future contributions.